kolibri/core/assets/src/composables/useUser.js

```diff
@@ -2,13 +2,13 @@ import { computed } from 'kolibri.lib.vueCompositionApi';
 import store from 'kolibri.coreVue.vuex.store';
 
 export default function useUser() {
-  const isUserLoggedIn = computed(() => store.getters.isUserLoggedIn);
-  const currentUserId = computed(() => store.getters.currentUserId);
-  const isLearnerOnlyImport = computed(() => store.getters.isLearnerOnlyImport);
-  const isCoach = computed(() => store.getters.isCoach);
-  const isAdmin = computed(() => store.getters.isAdmin);
-  const isSuperuser = computed(() => store.getters.isSuperuser);
-  const canManageContent = computed(() => store.getters.canManageContent);
+  const isLearnerOnlyImport = computed(() => store.getters['session/isLearnerOnlyImport']);
```

Comment:
This is not what was intended at all.

The purpose of the issue is to continue the work that was started here, providing composable interface to each of the getters in the session module, and to each element of the store state.

No changes to these existing exports should be made, and no changes should be made to session.js.

Instead, I should be seeing additional computed props returned from this composable that map to each of the getters defined in session.js and to each of the state values defined in the same file.